go/cmd/dolt/commands/sql.go

```diff
@@ -455,13 +455,21 @@ func execQuery(
 		return errhand.VerboseErrorFromError(err)
 	}
 
+	returnFormat := se.GetReturnFormat()
```

Comment:
I actually want you to use a set of regexes and back track down to the correct returnFormat